### PR TITLE
{bio}[foss-2016b] segemehl 0.2.0

### PR DESCRIPTION
**Pull request by SethosII**
Adds segemehl 0.2.0 with foss toolchain.

### CODE REVIEW

**Comment by SethosII** [comment]
The test fails because there is no `ncurses-5.9-foss-2016b`. I somehow have a `ncurses-5.9-foss-2016b` in my `ebfiles_repo` (and a working module from this). But `segemehl` also seems to work with `ncurses-6.0`, so changing the dependency might be better?

**Comment by boegel** [comment]
@SethosII Yes, please use `ncurses` 6.0 when using `foss/2016b`

File: easybuild/easyconfigs/s/segemehl/segemehl-0.2.0-foss-2016b.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+dependencies = [
+    ('ncurses', '5.9'),
+    ('zlib', '1.2.8'),
+]
+
+files_to_copy = [(["lack.x", "segemehl.x", "testrealign.x"], "bin")]
+
+sanity_check_paths = {
+    'files': ["bin/%s" % x for x in ["lack.x" , "segemehl.x", "testrealign.x"]],
+    'dirs': ["."]
```
Comment (boegel) [inline easybuild/easyconfigs/s/segemehl/segemehl-0.2.0-foss-2016b.eb]: @SethosII Please use an empty list here instead (`[]`)

**Comment by SethosII** [comment]
@boegel I added the requested changes.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2423.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/980bbbe6176cbee974a6391e4dee64e9 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2056.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/4efd3c00a531ea6145af6a5cdde26edf for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Going in, thanks @SethosII!

**Pull request merged**